MyCryptoHQ/MyCrypto
# Trezor v7 + CSP Fix

Closes #2394

### Description

Updates Trezor Connect to v7 and fixes CSP for `https://connect.trezor.io`

### Changes

* Updated TrezorConnect to v7
* Changed TrezorConnect import type to package.
* Fixes CSP for TCv7

### Steps to Test

1. Access Trezor Wallet
2. Try to send 0 ETH to yourself.
3. Try to display address.

~TODO: Test Electron Build~ - Electron build appears good to go : - 0
